src/ccnl-pkt/src/ccnl-pkt-builder.c

```diff
@@ -238,22 +238,18 @@ ccnl_mkInterestObject(struct ccnl_prefix_s *name, int *nonce)
 }
 
 struct ccnl_buf_s*
-ccnl_mkSimpleInterest(struct ccnl_prefix_s *name, int *nonce)
+ccnl_mkSimpleInterest_scratch(unsigned char *scratch, int scratch_len,
```

Comment:
Don't get me wrong, but the different naming schemes puzzle me (and I am aware that you just using/extending existing code). Why is this a mixture of camel case mixed with a good old fashion underscore C module naming? 

What I came up with is that we use "ccnl_" for functions and "ccn_lite" for code which is related to shell commands and "ccnl_mkBla..." for the actual command? Can somebody confirm this? Maybe is this documented somewhere?

Reply:
This is correct.